Snippets/ASB/ASB_6/Usage.cs

```diff
@@ -13,6 +13,17 @@ class Usage
         #endregion
     }
 
+    void NamespaceRoutingSendOptions(IBus bus)
+    {
+        string destination;
+        #region namespace_routing_send_options_full_connectionstring
+
+        destination = "sales@Endpoint=sb://destination1.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]";
+        bus.Send(destination, new MyMessage());
```

Comment:
In that case, would be nice if we could have a sample that use one way, but the text linked to the all possible ways (if that doco exists).